hostdiscovery/hostdiscovery_test.go

```diff
@@ -48,6 +48,9 @@ func TestAddedCallback(t *testing.T) {
 	if !added {
 		t.Fatal("Expecting an Added callback notification. But none received")
 	}
+	if removed {
+		t.Fatalf("Not expecting an Removed callback notification. But received a callback")
```

Comment:
You do not need to use `Fatalf()` here - you can use `Fatal()`

Reply:
Thank you, I'm on it.